extensions/eclipse-che-theia-plugin-remote/src/node/execute-command-container-aware.ts

```diff
@@ -8,18 +8,18 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 import { CommandRegistryImpl } from '@theia/plugin-ext/lib/plugin/command-registry';
-import { overrideUri } from './che-content-aware-utils';
+// import { overrideUri } from './che-content-aware-utils';
```

Comment:
@vparfonov a lot of commented lines in the file. Do we need them?

Reply:
just testing, will remove